R/vignettes.R

```diff
@@ -34,20 +34,27 @@
 
   conversion_worked <- vector(length = n)
 
+  fs::dir_copy(vignettes_path, articles_path)
+  vignettes_path2 <- paste0(articles_path, "/vignettes/")
```

Comment:
There are several examples of this, and it may be unsafe on some platforms. Switch all `paste0` path construction to this pattern: `vignettes_path2 <- fs::path_join(c(articles_path, "/vignettes/"))`